### PR TITLE
 Issue #SB-7869 feat: Border of content player and Height issue solved

### PR DESCRIPTION
**Pull request by ttplsudhir**

### CODE REVIEW

**Comment by ttplsudhir** [comment]
@lakhanmandloi - Please review PR

File: src/app/client/src/app/modules/workspace/components/upforreview-contentplayer/upforreview-contentplayer.component.html [hunk 1]
```diff
@@ -31,7 +31,7 @@ <h5 class="ui right floated basic icon circular button margin-auto mouse-pointer
     </div>
 
     <div class="row margin-top-10" *ngIf="!showLoader">
-      <div class="margin-top--5 videosegment ui clearing segment">
+      <div class="margin-top-20 videosegment ui clearing segment">
```
Comment (lakhanmandloi) [inline src/app/client/src/app/modules/workspace/components/upforreview-contentplayer/upforreview-contentplayer.component.html]: @ttplsudhir Replace classes ```margin-top-``` with ```mt-```

Comment (ttplsudhir) [reply src/app/client/src/app/modules/workspace/components/upforreview-contentplayer/upforreview-contentplayer.component.html]: @lakhanmandloi - Done.

**Comment by ttplsudhir** [comment]
**Before UI change**
contentplayer_beforeboardershadow
**After UI change**
contentplayer_afterboardershadow

**Pull request merged**